pages/testnets/testnet_info.md

```diff
@@ -8,7 +8,7 @@ The above info can be found in this [`networks` repository](https://github.com/d
 | ---------------- | --------------- | ------------- | -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
 | `dydx-testnet-1` | Test Network #1 | 🔴 Inactive    | v0.0.1-rc1     |                                                                                                                                                                                                                                                                                                                       | [Link](https://github.com/dydxprotocol/networks/tree/main/dydx-testnet-1) |
 | `dydx-testnet-2` | Test Network #2 | 🔴 Inactive    | v0.1.0-rc2     |                                                                                                                                                                                                                                                                                                                       | [Link](https://github.com/dydxprotocol/networks/tree/main/dydx-testnet-2) |
-| `dydx-testnet-3` | Test Network #3 | 🏗️ Preparation | v0.2.1         | 5b00f9ab668c35f7fcaff9a0607da59273bee399@dydx-testnet3-seednode.allthatnode.com:26656, 5454e22c769c5103e51c336121c532e9d6289348@tenderseed.ccvalidators.com:29103, f3339d67eac6e6a082555d2db6556ee4c0ce5f61@test-dydx-seed.kingnodes.com:26856, 20e1000e88125698264454a884812746c2eb4807@seeds.lavenderfive.com:23856 | [Link](https://github.com/dydxprotocol/networks/tree/main/dydx-testnet-3) |
+| `dydx-testnet-3` | Test Network #3 | 🏗️ Preparation | v0.2.1         | 5b00f9ab668c35f7fcaff9a0607da59273bee399@dydx-testnet3-seednode.allthatnode.com:26656, 5454e22c769c5103e51c336121c532e9d6289348@tenderseed.ccvalidators.com:29103, f3339d67eac6e6a082555d2db6556ee4c0ce5f61@test-dydx-seed.kingnodes.com:26856, 20e1000e88125698264454a884812746c2eb4807@seeds.lavenderfive.com:23856, 87ee8de5f0f82af6ee6740a30f8844bbe6434413@seed.dydx-testnet.cros-nest.com:26656 | [Link](https://github.com/dydxprotocol/networks/tree/main/dydx-testnet-3) |
```

Comment:
you should also update these two other lines in the `starting the network`:
1. https://github.com/dydxprotocol/v4-documentation/blob/dc441861646f03deae3fcf0c82d9343be5963673/pages/validators/starting_the_network.md?plain=1#L110
2. https://github.com/dydxprotocol/v4-documentation/blob/dc441861646f03deae3fcf0c82d9343be5963673/pages/validators/starting_the_network.md?plain=1#L139